support-frontend/assets/helpers/tracking/acquisitions.js

```diff
@@ -303,7 +303,7 @@ function getAcquisitionDataFromPPCParams(): ?Object {
     return {
       source: 'PPC',
       campaignCode: 'Google_Adwords',
-      queryParameters: getAllQueryParams(),
+      queryParameters: toAcquisitionQueryParameters(getAllQueryParams()),
```

Comment:
we still need to know whether to add the exclusions to this
https://github.com/guardian/support-frontend/blob/7e5384152a8fe6168126fd44b9df445e5ab8051f/support-frontend/assets/helpers/tracking/acquisitions.js#L176
hope @paulbrown1982 can help with this decision